commands/upload.ts

```diff
@@ -218,7 +218,13 @@ exports.handler = async options => {
       //  If clean is true, will first delete the dest folder and then upload src. Cleans up files that only exist on HS.
       let cleanUpload = options.force;
       if (!options.force) {
-        cleanUpload = await cleanUploadPrompt(derivedAccountId, dest);
+        cleanUpload = await confirmPrompt(
```

Comment:
Nice 👍 I'm a fan of cutting down on prompt utils that can be replaced by confirmPrompt()